playground/header.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
+* {box-sizing: border-box;}
+
+body { 
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.header {
+  overflow: hidden;
+  background-color: #f1f1f1;
+  padding: 20px 10px;
+}
+
+.header a {
+  float: left;
```

Comment:
Please use flexbox to keep things responsive, although float works but not on all the devices the its consistent

- [ ] Use flexbox

Reply:
I have to learn about flexbox .So it will take some time

Reply:
No problem @SwatiRaoDataEnthusiast Take your time.
Here is a useful link to learn about flexbox
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
I think for first pr after she split it up we should approve.

Flexbox can be next PR. 

I agree to encourage to use Flexbox.